src/IAstriaWithdrawer.sol

```diff
@@ -15,6 +17,15 @@ abstract contract IAstriaWithdrawer {
     // the denomination of the asset on the base chain.
     string public BASE_CHAIN_ASSET_DENOMINATION;
 
+    // The fee charged for each withdrawal transaction
+    uint256 public WITHDRAWAL_FEE;
+
+    // The address that receives the accumulated withdrawal fees
+    address public FEE_RECIPIENT;
+
+    // The total amount of fees collected & unclaimed from withdrawals
+    uint256 public ACCUMULATED_FEES;
```

Comment:
i would just use `address(this).balance` in `claimFees`

Reply:
This won't work for the native bridge tho as the amount being withdrawn just sits in the contract.